src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3701,8 +3701,8 @@ You should consider suppressing the warning only if you're sure that you don't w
   <data name="ERR_OverloadRefOutCtor" xml:space="preserve">
     <value>Cannot define overloaded constructor '{0}' because it differs from another constructor only on ref and out</value>
   </data>
-  <data name="ERR_ReturnNotAllowedInScript" xml:space="preserve">
-    <value>You cannot use 'return' in top-level script code</value>
+  <data name="ERR_YieldNotAllowedInScript" xml:space="preserve">
+    <value>You cannot use 'yield' in top-level script code</value>
   </data>
   <data name="ERR_NamespaceNotAllowedInScript" xml:space="preserve">
     <value>You cannot declare namespace in script code</value>
```

Comment:
We should consider removing "You" from the error messages. Perhaps just "Cannot use ..." and "Cannot declare ...".